taxi/views.py

```diff
@@ -104,14 +162,14 @@
 
 class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
     model = Driver
-    success_url = reverse_lazy("")
+    success_url = reverse_lazy("taxi:driver-list")
 
 
 @login_required
 def toggle_assign_to_car(request, pk):
     driver = Driver.objects.get(id=request.user.id)
     if (
-        Car.objects.get(id=pk) in driver.cars.all()
+            Car.objects.get(id=pk) in driver.cars.all()
     ):  # probably could check if car exists
```

Comment:
The comment suggests checking if the car exists, but the current implementation does not handle the case where the car with the given `pk` might not exist. Consider adding error handling to manage cases where the car does not exist, such as using `get_object_or_404`.